docs/metadata.rst

```diff
@@ -5,26 +5,33 @@ Metadata
 ========
 
 The tree-sequence and every entity within it (nodes, mutations, edges,  etc.) can have
-metadata associated with it. This is intended for storing and passing on information
+metadata associated with them. This is intended for storing and passing on information
 that tskit itself does not use or interpret. For example information derived from a VCF
 INFO field, or administrative information (such as unique identifiers) relating to
 samples and populations. Note that provenance information about how a tree sequence
 was created should not be stored in metadata, instead the provenance mechanisms in
 tskit should be used. See :ref:`sec_provenance`.
 
-The metadata for each entity is described by a schema for each entity type. This
-schema allows the tskit Python API to encode and decode metadata and, most importantly,
-tells downstream users and tools how to decode and interpret the metadata. This schema
-is in the form of a
-`JSON Schema <http://json-schema.org/>`_. A good to guide to creating JSON Schemas is at
+The metadata for each entity (i.e. row in a table) is described by a schema for each
+entity type (i.e. table). In this way every node's metadata follows the node schema,
+every mutation the mutation schema etc. Typically these schemas are defined by the software
+which created the tree-sequence file, as the exact metadata
+stored will vary depending on the use case. Subsequent processes can add or modify the schemas
+if they wish to add or modify to the types (or encoding) of the metadata. Most users of tree-sequence
+files will not need to modify the schemas.
```

Comment:
Perhaps add "The purpose of the schema is to say what information is stored in each metadata record, and how it is stored." - maybe after the first sentence?

Reply:
Added in https://github.com/tskit-dev/tskit/pull/741/commits/837e0e8b5474a6d69660ad1b0f30e236dc908417